OracleLinuxDevelopers/oraclelinux8/nodejs/14-oracledb/Dockerfile

```diff
@@ -1,15 +1,17 @@
 # Copyright (c) 2021 Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-FROM oraclelinux:8-slim
+FROM ghcr.io/djelibeybi/oraclelinux8-compat:8-slim
 
 COPY nodejs.module /etc/dnf/modules.d/nodejs.module
 
-RUN microdnf install oracle-instantclient-release-el8 oraclelinux-developer-release-el8 && \
-    microdnf install libaio \
-                     oracle-instantclient-basic \
-                     node-oracledb-node14 \
-                     && \
-    microdnf clean all
+RUN dnf -y install oracle-instantclient-release-el8 \
+                   oraclelinux-developer-release-el8 \
+                   && \
+    dnf -y install libaio \
```

Comment:
The IC Basic package has a dependency on libaio, so technically this explicit dependency isn't need.

Reply:
I've left the original Dockerfiles intact, i.e. I just want to change the base image. We can fix this afterwords.